src/control/cam_control_mod.meta

```diff
@@ -0,0 +1,27 @@
+[ccpp-table-properties]
+  name = cam_control_mod
+  type = module
+
+[ccpp-arg-table]
+  name  = cam_control_mod
+  type  = module
+[ eccen ]
+  standard_name = eccentricity_factor
+  units = 1
+  type = real | kind = r8
+  dimensions = ()
+[ obliqr ]
+  standard_name = obliquity
+  units = radian
```

Comment:
In some of the atmospheric physics code, the unit is plural - `radians`. Are both acceptable?

Reply:
nope! someday we might have a "conversion" between equivalent units of "radian" and "radians" but for now I've changed these to the plural version to match the rest of the code.

Thanks!